.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,3 +7,9 @@
  - [ ] Touched code is covered by tests
  - [ ] Documentation has been updated if necessary
  - [ ] [CLA](https://crate.io/community/contribute/cla/) is signed
+ - [ ] This does not contain breaking changes, or if it does:
+    - It is released within a major release
+    - It is recorded in ``CHANGES.txt``
```

Comment:
Maybe we can remove this since we've already a reminder about the changes?

Reply:
It is a bit redundant, but I'd keep it anyway